### ISSUE TITLE
Provide windows build?

### ISSUE THREAD

**Issue by yarikoptic** [conda-forge/skopeo-feedstock#9]
It seems (e.g. see https://github.com/containers/skopeo/issues/715) that it is generally possible and might work ;)

**Comment by mbargull** [comment]
We could try a cross-compile on Linux with `DISABLE_CGO=1`. Do you have access to a Windows machine to tests resulting builds then?
I could try to prepare a PR from which you could then build a package locally (on WSL or a separate Linux build system) and whose resulting package you could try out on a Windows host.
(If you want to work yourself on how to build on Windows natively, please let me know. I can only offer to try out said cross-compilation and am not able to test the resulting package.)

**Comment by yarikoptic** [comment]
Is cross compiling supported on conda forge?

I can test locally - windows vm is paused but the life is such that I do unpause it a few times a day

**Comment by mbargull** [comment]
> Is cross compiling supported on conda forge?

Yes. I wouldn't advice to go out and try to cross-compile a lot for Windows because we'd have to be careful in terms of compatibility/interaction with other packages (which are usually compiled with MSVC) -- and because I have no clue on who complicated this can get. But if we are talking about a static Go binary that doesn't link to anything (including a C library), we can give it a go ;).

> I can test locally - windows vm is paused but the life is such that I do unpause it a few times a day

Perfect!

**Comment by mbargull** [comment]
As it turned out, all previous cross-compiles have been with different "sudirs" but always on the same platform (Linux on Linux or macOS on macOS). If we want "Windows on Linux", we need https://github.com/conda/conda-build/pull/4124 to be added to `conda-build` first.
